apps/docs/src/content/docs/explainers/task-modifiers.mdx

```diff
@@ -3,4 +3,199 @@ title: Task modifiers
+`perform` button and reset the demo with the `clear timeline` button. The moment you perform a task,
+the timeline will start and it will show you the state of the task. Each task will run for two
+seconds and you can either click on it individually to cancel it or click on the `cancelAll` button
+to nuke them all!
+
+<Timeline show_max={false} client:load />
+
+## Enqueue
+
+The `enqueue` task, as the name suggests, creates a queue of all the task instances after the max
+concurrency has been reached. This means that up until the moment max concurrency is achieved every
+task instance will be executed immediately. But if you try to perform a new task instance and the
+number of running instances is at the `max` level, that new task instance will be stored in a queue
+and executed as soon as the number of concurrent instances drops below the `max` again.
+
+This is the right kind of task to use when you want to be sure that every task instance is executed
+at some point (unless the user leaves the page) but you also don't want to run them concurrently. A
+good use case for this could be uploading a list of files to the server. You want to upload all of
+your files but you don't want to send them all together because that would be too resource intensive
+and also makes it harder to handle in case of an error. You can use `enqueue` to allow only `max`
+file uploads at a time.
+
+```svelte
+<script lang="ts">
+	import { task, timeout } from '@sheepdogs/svelte';
+	import upload from './file-upload';
+
+	let files = [];
+
+	const uploadFile = task.enqueue(
+		async (file: File) => {
+			await upload(file);
+		},
+		{ max: 3 },
+	);
+</script>
+
+<input
+	type="file"
+	multiple
+	on:change={(e) => {
+		files = [...e.target.files];
+	}}
+/>
+<button
+	on:click={() => {
+		for (let file of files) {
+			uploadFile.perform(file);
+		}
+	}}>start upload</button
+>
+```
+
+You can play around with it in our Timeline and you may notice a new parameter available for you:
+initially set at 1, you can update the `max` parameter to allow for more task instances to run at
+the same time.
+
+<Timeline selected_task_type="enqueue" client:load />
+
+## Drop
+
+The `drop` task will simply ignore (and thus drop) every `perform` that is executed when the `max`
+amount of concurrent task instances is reached. In our example with a max concurrency of 1 for two
+seconds after the first perform no other task instance will be executed.
+
+You can think of this kind of task as a sort of throttle; and since the function will be just tossed
+away you should use this modifier only when you don't care about the "execution loss". A good
+example could be sending your mouse position to a websocket to be broadcasted to all the other
+users. You don't want to update it too often to avoid clogging your server, you don't care about the
+middle positions being lost, and the final mouse position is the most important.
```

Comment:
In this specific example, I would have thought `keepLatest` would be the best option.